spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/data/couchbase/CouchbaseReactiveRepositoriesAutoConfigurationTests.java

```diff
@@ -116,7 +115,8 @@ private void load(Class<?> config, String... environment) {
 
 	@Configuration
 	@TestAutoConfigurationPackage(CouchbaseReactiveRepositoriesAutoConfigurationTests.class)
-	@EnableMongoRepositories(basePackageClasses = CityCouchbaseRepository.class)
+	@EnableReactiveCouchbaseRepositories(basePackageClasses = ReactiveCityCouchbaseRepository.class)
```

Comment:
While the copy/pasted annotation there was wrong, I think the intend was to `@EnableCouchbaseRepositories` and check that no reactive repositories were enabled. Changing that  breaks that test so we need to investigate a bit more what is going on.